images/testing/ssh-executor/ssh.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Supposed to be added to the image?

Reply:
it added via bazel file
```
pkg_tar(
    name = "ssh_executor_tar",
    srcs = [
        ":ssh.sh",
        ":vagrant.key",
    ],
    package_dir = "/usr/local/bin",
    modes = {
        "ssh.sh": "0755",
        "vagrant.key": "0644",
    },
)
 container_image(
    name = "ssh_executor_image",
    base = ":passwd_image",
    tars = [":ssh_executor_tar"],
    entrypoint = ["/usr/local/bin/ssh.sh"],
    user = "ssh-executor",
    visibility = ["//visibility:public"],
)
```
Currently `container_image` does not support installation of RPM's, so first I create base image with all needed packages and after it use this image under `container_image` via `base = "@openssh_client//image"`. I am not happy with this, but @rmohr already added some RPM's support - https://github.com/rmohr/rules_docker/commit/3e65d3121bf05bfdb93e3c2642251aafdc1502c1
Also, maybe this separation does not so bad, because it gives possibility to monitor exact version for RPM's that our image has.

Reply:
The rules are now hosted here. Still pretty basic and only adds a `rpm_container` rule. Some rules to make  installing many RPMs more convenient are still missing.